### ISSUE TITLE
Getting user's ingredients from database

### ISSUE THREAD

**Issue by JoshuaRaybould** [JoshuaRaybould/SSH#58]
I am now going to begin creating a function that will take a users id and return their ingredients (name, quantity and quality) so that this information can then be used to match recipes best suited to the user.

**Comment by varsityyy** [comment]
Sounds good!

**Comment by JoshuaRaybould** [comment]
I've done this and created a pull request, but I think I could improve the testing so we can check values fall into certain ranges for randomly generated user ingredients once that is done.

**Comment by JoshuaRaybould** [comment]
I'm planning to look into better testing now, potentially using mockito as suggested by @varsityyy. I think I'll also use this as an opportunity for general clean up.

**Comment by JoshuaRaybould** [comment]
In fact I will just focus on testing after the clean up which I will make a new issue for now.
